packages/cardano-services/src/Projection/prepareTypeormProjection.ts

```diff
@@ -347,7 +382,9 @@ export const prepareTypeormProjection = (
     entities: selectedEntities,
     extensions,
     mappers: selectedMappers,
-    stores: selectedStores
+    stores: selectedStores,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    willStore: <T>(evt: T) => willStoreCheckers.some((check) => check(evt as any))
```

Comment:
Please add some test(s) for this, doesn't have to be exhaustive

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/f0d303377b97a236077dec1df3dce5eebe228aef , let me know if this is sufficient

Reply:
Thanks, looks good